SingularityUI/app/utils.es6

```diff
@@ -228,6 +228,17 @@ const Utils = {
     return splits.slice(0, splits.length - 5).join('-');
   },
 
+  getListOfUniqueRequestsFromListOfTasks(listOfTasks) {
+    const res = [];
+    for (const TaskId in listOfTasks) {
+      requestId = TaskId.requestId;
```

Comment:
nit: we generally want `UpperCamelCase` for classes and `lowerCamelCase` for instances.